universal-application-tool-0.0.1/app/services/cloud/azure/BlobStorage.java

```diff
@@ -71,13 +71,18 @@ public BlobStorage(Credentials credentials, Config config, Environment environme
   }
 
   @Override
-  public URL getPresignedUrl(String fileName, Optional<String> prefixedOriginalFileName) {
+  public String getPresignedUrlString(String fileKey) {
+    return getPresignedUrlString(fileKey, /* prefixedOriginalFileName= */ Optional.empty());
+  }
+
+  @Override
+  public String getPresignedUrlString(String fileName, Optional<String> prefixedOriginalFileName) {
```

Comment:
nit (super-minor): should we decide on either fileKey or fileName for this variable name? both are the same thing, right?

Reply:
Good idea! I'm going to go with fileKey